src/main/java/com/mycompany/app/App.java

```diff
@@ -11,9 +11,21 @@ public static void main( String[] args )
         System.out.println( "Hello World!" );
 
         // Generate some findbugs perhaps
-        int j;
+        String a = "hello";
+		String b = "goodbye";
+
+		if( a == b )
+		{
+			System.out.println( "Hello World!" );
+		}
+
+		String aString = "bob";
+		  b.replace('b', 'p');
```

Comment:
Reporter: FINDBUGS
Rule: RV_RETURN_VALUE_IGNORED
Severity: ERROR
File: com/mycompany/app/App.java L23
 Source: com.mycompany.app.App


Method ignores return value


   <p> The return value of this method should be checked. One common
cause of this warning is to invoke a method on an immutable object,
thinking that it updates the object. For example, in the following code
fragment,</p>
<blockquote>
<pre>
String dateString = getHeaderField(name);
dateString.trim();
</pre>
</blockquote>
<p>the programmer seems to be thinking that the trim() method will update
the String referenced by dateString. But since Strings are immutable, the trim()
function returns a new String value, which is being ignored here. The code
should be corrected to: </p>
<blockquote>
<pre>
String dateString = getHeaderField(name);
dateString = dateString.trim();
</pre>
</blockquote>


<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>